src/NLog/Targets/TraceTarget.cs

```diff
@@ -62,6 +63,12 @@ namespace NLog.Targets
     public sealed class TraceTarget : TargetWithLayout
     {
         /// <summary>
+        /// Always use <see cref="Trace.WriteLine(string)"/> independent of <see cref="LogLevel"/>
+        /// </summary>
+        [DefaultValue(false)]
+        public bool WriteLine { get; set; }
```

Comment:
I'm doubting if this is a good name. With WriteLine false I would expect Write without newline (for example in the stringBuilder)

Reply:
What about: writeMessageOnly, or rawWrite?

Reply:
Now it is valled RawWrite